### PR TITLE
update web bubble for credentials

### PR DESCRIPTION
**Pull request by usamabinnadeem-10**
## Done

- Update the web bubble for credentials

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Go to [this](https://docs.google.com/spreadsheets/d/1Ku_k8G_YssB_XPP3yyOSq8eG2THheEu3jBvWWv0RMZQ/edit?gid=362118719#gid=362118719) spreadsheet for suggested UI changes

## Issue / Card

Fixes [#14502](https://warthogs.atlassian.net/browse/WD-14502)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by linux-in-flux** [comment]
credentials page: 
- My understanding of Ege's note was to not use an exclamation point here, and she preferred "Get the exam" to  "Take the exam". I would also suggest this, as they are purchasing it but not necessarily taking it immediately.
- Still has "Get involved" section although button was merged with above section.

Syllabus page

- should be renamed to "Exam Content" (this was in a comment, but I don't think Ege updated the document.)
- Top copy should be shortened per E2
- Should have "Get Certification" button per E2
- Subitems should be bullet points instead of numbers (see E3)
- Check with Ege, as the 

Self-Study (aka Study Guide):
- Tabs have been made horizontal but don't match style from Syllabus page as requested

FAQ:
- Sidebar links are shortened but section titles are not. Clarify intention with Ege.
- Width is still inconsistent at "Before..." Transition (see E4)

Your-exams E3 is unverified because I have a million exams and don't know what it looks like w/o.

Shop

- Coming Soon needs to be corrected to Coming soon (no capitalization), check with Ege on exclamation point. She didn't say no here, but she said to be strict with exclamation point usage.

Checkout

- D3 was a little unclear, but I believe what is being asked is to remove "* indicates a mandatory field" from the top and put it underneath each mandatory field. Also, Region should be marked as a mandatory field. Note that this would also impact Ubuntu Pro, so you may want to check with them, but it sounded like she already talked to them?

**Comment by linux-in-flux** [comment]
For the "Get involved" section, the design review says "We can merge the section below [Get involved] to this one [SME/Tester], it doesn't need its section. The amount of space it occupies isn't justified against the amount of information it conveys." So yes, the copy changes made in the SME section and addition of Contact Us button make the Get involved section redundant, please remove.

"Exam content": 
- Across bubble, change navigation button.
- On credentials main page change links
- On [currently Syllabus page] change titles at top of page and <title>.
Eliminate the word "Syllabus" from bubble due to translation issues, per Cedric.

**Comment by linux-in-flux** [comment]
Still has Get Involved section. The contact button has been merged above. Please remove it.

Width is still wrong on FAQ at "Before your exam." Please make <ul> the same restricted width as your paragraphs per original instructions.

There are still many remaining issues that I assume are pending Ege. Have you reached out to her?

**Comment by linux-in-flux** [comment]
Screenshot from 2024-09-26 07-56-04

Screenshot from 2024-09-26 07-59-53

File: templates/credentials/index.html [hunk 1]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
-              </div>
-              <div class="col-2">
-                <a href="/credentials/shop">Take now&nbsp;›</a>
-              </div>
+        <div class="col-3"><h2 class="u-sv2">Complete Syllabus</h2></div>
+      </div>
+      <div class="row">
+        <div class="col-9 col-start-large-4">
+          <div class="row">
+            <div class="col-3" style="display:flex;flex-direction:column;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:120]:
```suggestion
            <div class="col-3">
```

File: templates/credentials/index.html [hunk 2]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <h3 class="p-heading--5 u-no-margin--bottom" style="height:4rem">CUE.01 Linux Quick Certification (QC)</h3>
+              <p class="u-no-padding--top">
+                Covers basic operational knowledge of Linux on security, operation and maintenance of basic system resources.
+              </p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
```
Comment (anthonydillon) [inline templates/credentials/index.html:125]:
```suggestion
              <hr />
```

File: templates/credentials/index.html [hunk 3]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <p style="margin-top:0.5rem;height:2rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:126]:
```suggestion
              <p>
```

File: templates/credentials/index.html [hunk 4]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+                <a href="/credentials/sign-up"
+                   class="p-button--positive u-no-margin--bottom">Get Certification</a>
+                <a href="/credentials/syllabus?exam=CUE.01%20Linux"
+                   class="p-button--link">Syllabus&nbsp;›</a>
+              </p>
+            </div>
+            <div class="col-3" style="display:flex;flex-direction:column;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:133]:
```suggestion
            <div class="col-3">
```

File: templates/credentials/index.html [hunk 5]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <h3 class="p-heading--5 u-no-margin--bottom" style="height:4rem">CUE.02 Desktop Quick Certification (QC)</h3>
+              <p class="u-no-padding--top">
+                Covers administrative essentials of Ubuntu Desktop such as package management, system installation and more.
+              </p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
```
Comment (anthonydillon) [inline templates/credentials/index.html:138]:
```suggestion
              <hr />
```

File: templates/credentials/index.html [hunk 6]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <p style="margin-top:0.5rem;height:2rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:139]:
```suggestion
              <p>
```

File: templates/credentials/index.html [hunk 7]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+                <a href="/credentials/syllabus?exam=CUE.02%20Desktop"
+                   class="p-button--link">Syllabus&nbsp;›</a>
```
Comment (anthonydillon) [inline templates/credentials/index.html:141]:
```suggestion
                   class="p-button--link">Syllabus<span class="u-off-screen"> for CUE.02 Desktop</span>&nbsp;&rsaquo;</a>
```

File: templates/credentials/index.html [hunk 8]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <h3 class="p-heading--5 u-no-margin--bottom" style="height:4rem">CUE.01 Linux Quick Certification (QC)</h3>
+              <p class="u-no-padding--top">
+                Covers basic operational knowledge of Linux on security, operation and maintenance of basic system resources.
+              </p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
+              <p style="margin-top:0.5rem;height:2rem;">
+                <a href="/credentials/sign-up"
+                   class="p-button--positive u-no-margin--bottom">Get Certification</a>
+                <a href="/credentials/syllabus?exam=CUE.01%20Linux"
+                   class="p-button--link">Syllabus&nbsp;›</a>
```
Comment (anthonydillon) [inline templates/credentials/index.html:130]:
```suggestion
                   class="p-button--link">Syllabus<span class="u-off-screen"> for CUE.01 Linux</span>&nbsp;&rsaquo;</a>
```

File: templates/credentials/index.html [hunk 9]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              </p>
+            </div>
+            <div class="col-3" style="display:flex;flex-direction:column;">
+              <h3 class="p-heading--5 u-no-margin--bottom" style="height:4rem">CUE.03 Server Quick Certification (QC)</h3>
+              <p class="u-no-padding--top">Covers common Ubuntu Server administrative tasks and troubleshooting.</p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
+              <p style="margin-top:0.5rem;height:2rem;">
+                <a href="/credentials/syllabus?exam=CUE.03%20Server"
+                   class="p-button--link">Syllabus&nbsp;›</a>
```
Comment (anthonydillon) [inline templates/credentials/index.html:150]:
```suggestion
                   class="p-button--link"><span class="u-off-screen"> for CUE.03 Server</span>&nbsp;&rsaquo;</a>
```

File: templates/credentials/index.html [hunk 10]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <p style="margin-top:0.5rem;height:2rem;">
+                <a href="/credentials/syllabus?exam=CUE.02%20Desktop"
+                   class="p-button--link">Syllabus&nbsp;›</a>
+              </p>
+            </div>
+            <div class="col-3" style="display:flex;flex-direction:column;">
+              <h3 class="p-heading--5 u-no-margin--bottom" style="height:4rem">CUE.03 Server Quick Certification (QC)</h3>
+              <p class="u-no-padding--top">Covers common Ubuntu Server administrative tasks and troubleshooting.</p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
+              <p style="margin-top:0.5rem;height:2rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:148]:
```suggestion
              <p>
```

File: templates/credentials/index.html [hunk 11]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <p class="u-no-padding--top">
+                Covers administrative essentials of Ubuntu Desktop such as package management, system installation and more.
+              </p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
+              <p style="margin-top:0.5rem;height:2rem;">
+                <a href="/credentials/syllabus?exam=CUE.02%20Desktop"
+                   class="p-button--link">Syllabus&nbsp;›</a>
+              </p>
+            </div>
+            <div class="col-3" style="display:flex;flex-direction:column;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:144]:
```suggestion
            <div class="col-3">
```

File: templates/credentials/index.html [hunk 12]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <h3 class="p-heading--5 u-no-margin--bottom" style="height:4rem">CUE.03 Server Quick Certification (QC)</h3>
+              <p class="u-no-padding--top">Covers common Ubuntu Server administrative tasks and troubleshooting.</p>
+              <hr class="is-fixed-width u-no-margin--bottom" style="margin-top:auto;" />
```
Comment (anthonydillon) [inline templates/credentials/index.html:147]:
```suggestion
              <hr />
```

File: templates/credentials/index.html [hunk 13]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
         <div class="col-6">
-          <p>Please click the button below to:</p>
-          <ul>
-            <li>
-              Receive exclusive access to preview, test, and critique exams in
-              development
+          <p>Register to be a pre-release tester or subject matter expert.</p>
+          <p class="u-no-margin--bottom">You'll get to:</p>
+          <ul class="p-list--divided u-no-margin--bottom">
+            <li class="p-list__item is-ticked" style="padding-bottom:0.5rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:166]:
```suggestion
            <li class="p-list__item is-ticked">
```

File: templates/credentials/index.html [hunk 14]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <strong>Receive exclusive access to preview, test, and critique exams in
+                development</strong>
+              
+            </li>
+            <li class="p-list__item is-ticked" style="padding-bottom:0.5rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:171]:
```suggestion
            <li class="p-list__item is-ticked">
```

File: templates/credentials/index.html [hunk 15]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <strong>Participate in surveys to approve exam topics</strong>
+            </li>
+            <li class="p-list__item is-ticked" style="padding-bottom:0.5rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:174]:
```suggestion
            <li class="p-list__item is-ticked">
```

File: templates/credentials/index.html [hunk 16]
```diff
@@ -27,142 +26,162 @@ <h1>Canonical Ubuntu Essentials</h1>
...
+              <strong>Join advisory panels on the content and direction of future exams</strong>
+            </li>
+            <li class="p-list__item is-ticked" style="padding-bottom:0.5rem;">
```
Comment (anthonydillon) [inline templates/credentials/index.html:177]:
```suggestion
            <li class="p-list__item is-ticked">
```

**Review by anthonydillon — APPROVED** [review]
Almost all is ok but can you strip the inline styles as much as possible and rely on Vanilla for styling. Code and QA :+1:

**Pull request merged**